go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Jigsaw-Code/outline-apps
 
-go 1.21
+go 1.22
```

Comment:
Are there any go `1.22` features we are using in this PR? If not, maybe we can separate the PR since it's already pretty big.

Reply:
I thought there was, but I guess not. Reverted.